indico/modules/events/requests/templates/event_request_details.html

```diff
@@ -5,10 +5,19 @@
 
 {% set has_req = req and not req.state.name == 'withdrawn' %}
 {% set show_management = is_manager and has_req %}
+{% set button_title -%}
+    {%- if not req -%}
+        {% trans %}Send request{% endtrans %}
+    {%- elif req.can_be_modified -%}
+        {% trans %}Modify request{% endtrans %}
+    {%- else -%}
+        {% trans %}Send new request{% endtrans %}
+    {%- endif -%}
+{%- endset %}
 
 {% block title %}{% trans %}Services{% endtrans %}{% endblock %}
 {% block subtitle %}{{ definition.title }}{% endblock %}
-
+{% block description %}{% endblock %}
```

Comment:
this shouldn't be necessary; it inherits from an already-empty description block